IREC/IREC_PDD.tex

```diff
@@ -0,0 +1,383 @@
+The timeline is based largely around two important dates. The first is the hand-off of the external mechanical design to LI on November 1st which will provide time for LI
+to incorporate Hyperion into their own designs. The second important date is the competition itself. Although the specific day has not been announced yet, past competition dates
+suggest that it will be held during late June. The timeline is as follows:
+\begin{itemize}
+  \item September 1st: Begin narrowing down payload ideas using a decision matrix based on the SDL Payload Challenge scoring rubric
+  \item September 8th: Decide on the specific payload and make a plan of action for research and design
+  \item October 1st: Present a preliminary design with estimates for size, cost, and weight
+  \item November 1st: Mechanical design delivery to LI. No changes to the external frame can be made after this point
+  \item Semester's End: Final hardware design completed and materials ordered
+  \item April 1st: Integration testing of rocket and payload
+  \item April/May: Test launch of rocket and payload
+  \item June 22nd-24th: Intercollegiate Rocket Engineering Competition
+\end{itemize}
+
+This timeline provides SPEX with roughly one month until a preliminary design is created. After this, there will be another month to refine
+the design before the final size and interface details must be provided to LI. After this point, there will be more internal
+design and development for the remainder of the semester with a goal of purchasing all materials by the semester's end.
+If this timeline can be adhered to, then the entire spring semester and summer months leading up to the competition can be
+used for manufacturing, programming, integration, testing, and characterization.
+
+\subsection{Deliverables}
+\label{subsec:Deliverables}
+  % When all is said and done, what will you have to show for it?
+  % Examples: Hardware, software, poster, ImagineRIT demo, presentations, technical papers...
+In order to be compliant with the SDL Payload Challenge requirements, a poster is needed as part of the judging criteria.
+Other deliverables for this project include the open-source technical documentation such as circuit schematics, CAD drawings,
+and embedded software. The final deliverable is a functioning payload module conforming to the IREC SDL payload challenge
+standard with accompanying ground station hardware and operating documentation.
+
+\section{Externalities}
+\label{subsec:Externalities}
+  % Things not directly related to the work or outcomes, but related to the project as a whole.
+\subsection{Prerequisite Skills}
+\label{subsec:Prerequisite Skills}
+  % Which skills do team members need to have before work can start (not including skills that will be learned ``on the job'')?
+To participate in this endeavor, there are no prerequisite skills. However, for the project to succeed, there will need to be
```

Comment:
No prerequisite skills at all?  You have an electrical payload AND a mechanical payload riding in a cubesat form factor, and a few months to pull it off... I'm not convinced